packages/style-engine/src/styles/background/index.ts

```diff
@@ -52,6 +52,18 @@ const backgroundRepeat = {
 	},
 };
 
+const backgroundPosition = {
+	name: 'backgroundRepeat',
+	generate: ( style: Style, options: StyleOptions ) => {
+		return generateRule(
+			style,
+			options,
+			[ 'background', 'backgroundPosition' ],
```

Comment:
I know it's in the "gather feedback" stage. Would also be good to add unit tests to the TODO list for this change. Thank you!

Reply:
Updated the unit tests to include this one in https://github.com/WordPress/gutenberg/pull/58592/commits/d9d87ba23f4604f1e7520ddc6e50f1fbce36f395 👍